internal/e2e/rds_test.go

```diff
@@ -309,7 +309,7 @@ func TestEditIngressInPlace(t *testing.T) {
 				Name: "ingress_http",
 				VirtualHosts: []route.VirtualHost{{
 					Name:    "hello.example.com",
-					Domains: []string{"hello.example.com", "hello.example.com:80"},
+					Domains: []string{"hello.example.com", "hello.example.com:*"},
```

Comment:
perhaps copy the `domains` helper from `contour/route_test.go` and clean this up a little.

Reply:
Good idea, done.